content/source/docs/enterprise/private/vmware-setup-guide.html.md

```diff
@@ -166,7 +167,7 @@ template.
 The PTFE application is connect to the Postgres database via the Postgres URL.
 
 The PTFE application is connected to object storage via the S3 endpoint for the
```

Comment:
Is it okay to still say S3 endpoint here, or should we say "storage address" or something more generic?